app/src/components/bcgov/ButtonStyles.ts

```diff
@@ -0,0 +1,37 @@
+import { bcgov } from "../../constants/colours";
+import { normalFont } from "../../constants/fonts";
+
+const commonButtonStyle = {
+    ...normalFont,
+    borderRadius: '4px',
+    padding: '12px 24px',
+    textAlign: 'center',
+    textDecoration: 'none',
+    display: 'inline-block',
+    fontSize: '14px',
+    fontWeight: '700',
+    letterSpacing: '1px',
+    cursor: 'pointer',
+  '&:hover': {
```

Comment:
same feedback as the other PR

Reply:
Should be covered from the other PR. Will merge to update this.